src/Aspire.Dashboard/Components/Controls/Chart/DashpageCharts.razor

```diff
@@ -10,44 +10,27 @@
     <h3 title="@SelectedDashpage.Name">@SelectedDashpage.Name</h3>
 }
 
-<FluentGrid Spacing="3" AdaptiveRendering="true" Justify="JustifyContent.FlexStart" Class="dashpage-chart-grid">
+<section class="dashpage-chart-grid">
```

Comment:
Why is a section element instead of div?

Reply:
`section` gives a bit more semantics, suggesting it's a larger area of content. Similarly, `article` is used for each chart. This theoretically gives tools that inspect the DOM (such as screen readers) more info about the DOM. `div`s get used for non-semantic layout.

Reply:
I wouldn't worry about being semantically correct. We're using div everywhere for block content and span for inline content.

Reply:
It is preferred to use the semantically correct element. I would not change to div just for the sake of it

Reply:
We aren't using semantically correct elements in other places. It seems like a non-goal and makes it different from everywhere else in the app which is always to use div. Often being consistently correct is better than being technically correct.

But arguing over html elements is the least productive thing in the world so if you think this is best then go ahead.